agent/php_agent.c

```diff
@@ -87,6 +90,32 @@ zval* nr_php_get_zval_object_property(zval* object,
   return 0;
 }
 
+zval* nr_php_get_zval_base_exception_property(zval* exception,
+                                              const char* cname) {
+  char* name;
+  zend_class_entry* ce;
+
+  if ((0 == exception) || (0 == cname) || (0 == cname[0])) {
```

Comment:
If we're checking if a pointer is `NULL`, we should be explicit in using `NULL` as the comparison value, rather than `0`, which can easily be misinterpreted for a numerical value check rather than a `NULL` check. In the same way, if we're returning a pointer value, we should stick to the convention of returning an explicit `NULL` where applicable rather than a `0`.

Reply:
Good call, these lines are just copied from the similar function directly above on line https://github.com/newrelic/newrelic-php-agent/pull/825/files#diff-3ae60d50007d698aae55659ea75350561bf514cf5b585e8c86676f26aff6dd04R71

Will change both

Reply:
fixed in [18f7088](https://github.com/newrelic/newrelic-php-agent/pull/825/commits/18f708807d20e7056fcaa7d1f80a14a8a985878a)